4_processing_features/Plate1/extract_sc_features_cp_plate1.ipynb

```diff
@@ -1394,309 +1422,66 @@
        "  </tbody>\n",
        "</table>\n",
        "</div>"
       ],
       "text/plain": [
-       "   WellRow  WellCol well_position gene_name genotype\n",
-       "0        A        1            A1       NF1       WT\n",
-       "1        A        6            A6       NF1       WT\n",
-       "2        A        7            A7       NF1     Null\n",
-       "3        A       12           A12       NF1     Null\n",
-       "4        B        1            B1       NF1       WT\n",
-       "5        B        6            B6       NF1       WT\n",
-       "6        B        7            B7       NF1     Null\n",
-       "7        B       12           B12       NF1     Null\n",
-       "8        C        1            C1       NF1       WT\n",
-       "9        C        6            C6       NF1       WT\n",
-       "10       C        7            C7       NF1     Null\n",
-       "11       C       12           C12       NF1     Null\n",
-       "12       D        1            D1       NF1       WT\n",
-       "13       D        6            D6       NF1       WT\n",
-       "14       D        7            D7       NF1     Null\n",
-       "15       D       12           D12       NF1     Null\n",
-       "16       E        1            E1       NF1       WT\n",
-       "17       E        6            E6       NF1       WT\n",
-       "18       E        7            E7       NF1     Null\n",
-       "19       E       12           E12       NF1     Null\n",
-       "20       F        1            F1       NF1       WT\n",
-       "21       F        6            F6       NF1       WT\n",
-       "22       F        7            F7       NF1     Null\n",
-       "23       F       12           F12       NF1     Null\n",
-       "24       G        1            G1       NF1       WT\n",
-       "25       G        6            G6       NF1       WT\n",
-       "26       G        7            G7       NF1     Null\n",
-       "27       G       12           G12       NF1     Null\n",
-       "28       H        1            H1       NF1       WT\n",
-       "29       H        6            H6       NF1       WT\n",
-       "30       H        7            H7       NF1     Null\n",
-       "31       H       12           H12       NF1     Null"
+       "  WellRow  WellCol well_position gene_name genotype\n",
+       "0       C        6            C6       NF1       WT\n",
+       "1       C        7            C7       NF1     Null\n",
+       "2       D        6            D6       NF1       WT\n",
+       "3       D        7            D7       NF1     Null\n",
+       "4       E        6            E6       NF1       WT"
       ]
      },
-     "execution_count": 13,
+     "execution_count": 14,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
     "# Load platemap file\n",
-    "platemap_df2 = pd.read_csv(platemap_file2)\n",
-    "platemap_df2"
+    "platemap_file = pd.read_csv(platemap_file)\n",
```

Comment:
need to change the variable name (it's not a file!)

Reply:
@gwaybio 

I am not sure where the issue is, mainly cause I forgot the py files. Can you show me in the py file for this. I do not see this platemap_df2.

Reply:
See commit 347c191